data/assets/01.css

```diff
@@ -1,3 +1,30 @@
-body {
-  background-color: white;
-}
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;600&display=swap');
+
+/* VARIABLES USED IN OTHER CSS FILES */
+:root {
+  --theme-0: #202020;
+  --theme-1: #454545;
+  --theme-2: #858585;
+  --theme-3: #CCC;
+  --theme-4: #F5f5f5;
+  --white-0: #FFFFFF;
+  /* CSS HEX */
+  --blue-2: #134074ff;
+  --blue-1: #13315cff;
+  --blue-0: #0b2545ff;
+  --blue-3: #8da9c4ff;
+  --blue-4: #eef4edff;
+  --gray-1: #eeeeee;
+  --color-warning: #FF3300;
```

Comment:
I recall it was mentioned at the beginning that we should warn the user that the cached data is stale eg in case of last data update failed. Don't we want that anymore?
Adding subtask for view auto-update [https://clkentik.atlassian.net/browse/KNTK-272](https://clkentik.atlassian.net/browse/KNTK-272)
Adding subtask for small screen support [https://clkentik.atlassian.net/browse/KNTK-273](https://clkentik.atlassian.net/browse/KNTK-273)